pages/en/lb4/Application.md

```diff
@@ -124,6 +124,42 @@ In the above example:
 - injection calls for `repositories.widget` will be handled by a singleton
 instance of the `WidgetRepository` class.
 
+#### Components
+```ts
+app.component(MyComponent);
+app.component(RestComponent);
+```
+The `component` function allows binding of component constructors within
+your `Application` instance's context.
+
+For more information on how to make use of components,
+see [Using Components](Using-components.html).
+
+#### Controllers
+```ts
+app.controller(FooController);
+app.controller(BarController);
```

Comment:
does it mean `app.controller` no longer takes in an **array** of controller ctor?

Reply:
It never did in the first place. The change is being made to a WIP PR that got accidentally merged, and since that feature (registration of array of controllers) is no longer being implemented, I'm reverting the process.